### PR TITLE
Add options that are required for Agora

### PR DESCRIPTION
**Pull request by AlexeyRaga**

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -8,8 +10,9 @@ path=$(stack path --local-install-root)
 
 ${path}/bin/${exe} \
   --kafka-broker localhost:9092 \
-  --kafka-group-id local-group \
   --kafka-schema-registry http://localhost:8081 \
-  --commands-topic commands \
+  --kafka-group-id ${CLUB_NAME}--indexer-group \
```
Comment (oliverdaff) [inline run.sh]: Minor niggle but perhaps name add `submissions` to this name `${CLUB_NAME}--submissions-indexer-group` so we can index other types of files also.

Comment (oliverdaff) [inline run.sh]: Other than this 👍

Comment (AlexeyRaga) [inline run.sh]: Good point, will do

Comment (newhoggy) [inline run.sh]: I agree.

**Comment by newhoggy** [comment]
👍

**Pull request merged**